docs/tutorials/cpp.md

```diff
@@ -72,11 +89,25 @@ int main() {
 }
 ```
 
+## Compiling with the ICU4X header files
+
+The header files are shipped inside the crate, which Cargo has put somewhere on your system. You can find its location with
+```shell
+HEADERS=$(cargo metadata --format-version 1 | jq '.packages[] | select(.name == "icu_capi").manifest_path' | xargs dirname)
```

Comment:
won't this be the root folder not the headers folder?

Reply:
ah good catch